scripts/test

```diff
@@ -37,7 +43,12 @@ copy_logs(){
 run_tests()
 {
     pushd ./tests/validation
-    tox || [ `uname` == "Linux" ] && copy_logs && exit 1
+    if [ "${CATTLE_TEST_PARALLEL_THREADS}" -gt 1 ] 
```

Comment:
I had it this way to allow for the output format to be shown with test suite breakdown which is better than what we get when executing with "-n 1"